lockbox-ios/View/ItemDetailView.swift

```diff
@@ -202,3 +209,20 @@ extension ItemDetailView: UIGestureRecognizerDelegate {
         }
     }
 }
+
+extension ItemDetailView: UITableViewDragDelegate {
+    func tableView(_ tableView: UITableView, itemsForBeginning session: UIDragSession, at indexPath: IndexPath) -> [UIDragItem] {
```

Comment:
(this is probably the wrong place to put this comment)

There are telemetry events for tracking when the username or password are copied.  We should try to record the same or similar event for drag/drop operations, if possible.